src/content/docs/en/core-concepts/routing.mdx

```diff
@@ -293,32 +293,45 @@ if (!isLoggedIn(cookie)) {
 
 ## Route Priority Order
 
-It's possible for multiple routes to match the same URL path. For example each of these routes would match `/posts/create`:
+It's possible for multiple routes to match the same URL path. For example all of these routes could match `/posts/create`:
 
 <FileTree>
 - src/pages/
+  - [...slug].astro
   - posts/
     - create.astro
-    - [pid].astro
+    - [page].astro
+    - [pid].ts
     - [...slug].astro
 </FileTree>
 
-Astro needs to know which route should be used to build the page. To do so, it sorts them according to the following rules:
+Astro needs to know which route should be used to build the page. To do so, it sorts them according to the following rules in order:
 
-- Static routes without path parameters will take precedence over all other routes
-- Dynamic routes using named parameters take precedence over rest parameters
-- Pre-rendered dynamic routes take precedence over server dynamic routes
-- Rest parameters have the lowest priority
-- Endpoints always take precedence over pages
-- Ties are resolved alphabetically
+- More specific rotues (with more path segments) will take precedence over less specific routes. In the example above all routes under `/posts/` take precedence over `/[...slug].astro` at the root.
+- Static routes without path parameters will take precedence over dynamic routes. `/posts/create.astro` takes precedence over all the other routes in the example.
+- Dynamic routes using named parameters take precedence over rest parameters. `/posts/[page].astro` takes precedence over `/posts/[...slug].astro`.
+- Pre-rendered dynamic routes take precedence over server dynamic routes.
+- Endpoints take precedence over pages.
+- If all the above doesn't decide the order, routes are sorted alphabetically based on the default locale of your Node instalation.
```

Comment:
```suggestion
- Static routes without path parameters will take precedence over dynamic routes. E.g. `/posts/create.astro` takes precedence over all the other routes in the example.
- Dynamic routes using named parameters take precedence over rest parameters. E.g. `/posts/[page].astro` takes precedence over `/posts/[...slug].astro`.
- Pre-rendered dynamic routes take precedence over server dynamic routes.
- Endpoints take precedence over pages.
- If none of the rules above decide the order, routes are sorted alphabetically based on the default locale of your Node installation.
```

Some typo fixes here, and perhaps add "E.g." to the examples, or something along the lines of that?